spec/mem/atomic.md

```diff
@@ -7,15 +7,8 @@ For now atomicity is ignored; this will change in the future.
 pub struct AtomicMemory<M: Memory> {
     memory: M,
 
-    /// The Id of the current thread
-    current_thread: ThreadId,
     /// List of all memory access done by the active thread in the current step.
     current_accesses: List<Access>,
```

Comment:
Now we don't have "current" and "previous" any more, so let's find a better name for this field. Just `accesses` could work? Or `recent_accesses` or so?